examples/dt/osasinfra/dataplane-post-ceph.md

```diff
@@ -0,0 +1,77 @@
+# Configuring and deploying the post-Ceph dataplane
+
+## Assumptions
+
+- The pre-Ceph [dataplane](dataplane-pre-ceph.md) was already deployed and Ceph was manually installed afterwards
+
+## Initialize post-Ceph
+
+Switch to the "openstack" namespace
+```
+oc project openstack
+```
+Change to the osasinfra directory
+```
+cd architecture/examples/dt/osasinfra
+```
+Edit the [values.yaml](values.yaml) and [service-values.yaml](service-values.yaml)
+files to suit your environment.
+```
+vi values.yaml
+vi service-values.yaml
+```
+The ceph sections of [values.yaml](values.yaml) should have values like this.
+```yaml
+data:
+    ceph:
+        conf: $CONF
+        keyring: $KEY
+
+```
+Where the values of the variables above can be retrieved by
+running the following commands on the Ceph cluster.
+```shell
+CONF=$(cat /etc/ceph/ceph.conf | base64 -w 0)
+KEY=$(cat /etc/ceph/ceph.client.openstack.keyring | base64 -w 0)
+```
```

Comment:
customizing nova to use ceph is deliberately not included since nova ephemeral is using local disk. +1